internal/wasm/memory_test.go

```diff
@@ -979,3 +993,25 @@ func requireChannelEmpty(t *testing.T, ch chan string) {
 		// fallthrough
 	}
 }
+
+func sliceAllocator(min, cap, max uint64) experimental.MemoryBuffer {
+	return &sliceBuffer{make([]byte, min, cap), max}
+}
+
+type sliceBuffer struct {
+	buf []byte
+	max uint64
+}
+
+func (b *sliceBuffer) Free() {}
+
+func (b *sliceBuffer) Buffer() []byte { return b.buf }
+
+func (b *sliceBuffer) Grow(size uint64) []byte {
```

Comment:
Please put comments on them like any other places in the code base
```suggestion
// Free implements ...
func (b *sliceBuffer) Free() {}

// Buffer implements ...
func (b *sliceBuffer) Buffer() []byte { return b.buf }

// Grow implements ...
func (b *sliceBuffer) Grow(size uint64) []byte {
```